resolvers/event.js

```diff
@@ -304,10 +304,8 @@ const eventResolvers = {
       });
       return db.event.findByPk(args.id);
     },
-    deleteEvent: async (parent, args) => {
-      // Return status for destroy
-      // 1 for successful deletion, 0 otherwise
-      await Promise.all([
+    deleteEvent: async (parent, args) =>
+      Promise.all([
         db.eventAmbulances.destroy({
```

Comment:
destroy returns a Promise that always gets resolved with 1 on success or 0 otherwise (i.e. the promise is always “fulfilled” and won’t trigger a catch statement event when the deleting failed).

Note this means that there's no "error handling" for `db.eventAmbulances.destroy` and `db.eventHospitals.destroy`. We can implement it but I couldn't find a clean way to do it.